SIS_import/sis_upload.rb

```diff
@@ -106,15 +101,15 @@ def Canvas_API_POST(url, params, fileName)
 	begin
 		if (fileName != nil)
 			# upload the SIS zip file
-			response = RestClient.post  Addressable::URI.escape(url),
-																	{:multipart => true,
-	                                :attachment => File.new(fileName, 'rb')
-																	},
-																	{:Authorization => "Bearer #{$token}",
-																	:accept => "application/json",
-																	:import_type => "instructure_csv",
-																	:content_type => "application/zip",
-																	:verify_ssl => true}
+			response = RestClient.post Addressable::URI.escape(url),
+			                           {:multipart => true,
+			                            :attachment => File.new(fileName, 'rb')
+			                           },
+			                           {:Authorization => "Bearer #{$token}",
+			                            :accept => "application/json",
+			                            :import_type => "instructure_csv",
+			                            :content_type => "application/zip",
+			                            :verify_ssl => true}
```

Comment:
Are these indentations intended? Or is this a result of the IDE you were using to write the code?